test/multistream.test.js

```diff
@@ -140,10 +140,34 @@ test('supports custom levels', function (t) {
     customLevels: {
       foo: 35
     }
-  }, multistream([{level: 35, stream: stream}]))
+  }, multistream([{ level: 35, stream: stream }]))
   log.foo('bar')
 })
 
+test('supports pretty print', function (t) {
+  var stream = writeStream(function (data, enc, cb) {
+    t.isNot(data.toString().match(/INFO.*: pretty print/), null)
+    t.done()
+    cb()
+  })
+  var outStream = pino({
+    prettifier: require('pino-pretty'),
```

Comment:
I don't think you need to specify here (default is to require pino-pretty) 

(if it turns out that you do can you please require pino-pretty with other deps at the top then include here by reference